portfolio/src/main/webapp/scripts/server-dev.js

```diff
@@ -12,17 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var i = 0;
```

Comment:
rename the variable here please to make it more meaningful. also probably add a comment for it if needed to explain what's that for.

Reply:
Done.